### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by CikiMomogi**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,6 @@
 /.idea/
 .DS_Store
+
+#pnpm
+site/pnpm-lock.yaml
+#pnpm end
```
Comment (roj1512) [inline .gitignore:6]: I think these changes should be kept only on your side since we are neither using pnpm, nor we have suggested it anywhere.

Comment (CikiMomogi) [reply .gitignore:6]: I don't think it's a big deal to include them. That can make it easier for use to exclude the file every time we want to commit.

Comment (roj1512) [reply .gitignore:6]: Why would you need to exclude it every time you commit? Just don't include it, that's it.

Comment (CikiMomogi) [reply .gitignore:6]: Correct. Everytime i want to commit, i must exclude them manually.

Comment (roj1512) [reply .gitignore:6]: What? How is that possible?

Comment (roj1512) [reply .gitignore:6]: As far as I know, Git does not add any changes, from any file, unless you explicitly do it with `git add`. Do you always do `git add -A`?

Comment (CikiMomogi) [reply .gitignore:6]: Yup. I always do git add -A rather than add every files manually. But, it is not big deal thou. Should i remove this line?

Comment (roj1512) [reply .gitignore:6]: I don’t mind if it stays there. I was just curious about it. It doesn’t change anything for the end users.

Comment (CikiMomogi) [reply .gitignore:6]: Trust me. It will help me a lot. 😁

Comment (roj1512) [reply .gitignore:6]: Well, let’s keep it then :)

**Review by roj1512 — CHANGES_REQUESTED** [review]
We should add a preceding `-` or `_` to the names of the imported `.sass` files.

**Review by roj1512 — APPROVED** [review]
Thanks! Sorry for the delay.

**Pull request merged**